lib/nexpose/global_blackout.rb

```diff
@@ -0,0 +1,43 @@
+module Nexpose
+
+  class GlobalBlackout < APIObject
+    require 'json'
+    include JsonSerializer
+
+    # [Array] Blackout starting dates, times and duration for blackout periods.
+    attr_accessor :blackout
+
+    def initialize(blackout)
+      @global_blackout = Array(blackout)
```

Comment:
I think it should be something else. We have a GlobalBlackout object what does a blackout of a GlobalBlackout mean? is it the schedule of a GlobalBlackout? if so then it should be called schedule. That way it's GlobalBlackout.schedule as  oppose to GlobalBlackout.blackout.